source/jormungandr/tests/instant_system_new_default_routing_tests.py

```diff
@@ -205,3 +206,61 @@ def test_basic_ride_sharing(self):
 
         assert rsj_sections[2].get('type') == 'crow_fly'
         assert rsj_sections[2].get('mode') == 'walking'
+
+    def test_ride_sharing_with_pt(self):
+        """
+        test ridesharing_jouneys details
+        """
+        q = journey_basic_query + \
+            "&last_section_mode[]=walking" + \
+            "&first_section_mode[]=ridesharing" + \
+            "&ridesharing_speed=2.5" + \
+            "&debug=true"
+        response = self.query_region(q)
+        self.is_valid_journey_response(response, q, check_journey_links=False)
+
+        journeys = get_not_null(response, 'journeys')
+        assert len(journeys) == 2
+
+        #The first journey is direct ridesharing
```

Comment:
if this journey is not present without the `debug=true` it's OK, but we have to check it's there (and it's the only one) if we request with `max_ridesharing_duration_to_pt=0`